debian/control

```diff
@@ -199,7 +199,10 @@ Description: EndlessOS Kernel Team Development Tools
 
 Package: eos-tech-support
 Architecture: all
-Depends: ${misc:Depends}, ${eos:Depends}, usbutils
+Depends: ${misc:Depends},
+         ${eos:Depends},
+         usbutils,
+         ostree (>= 2018.8),
```

Comment:
Nitpick: One of my favourite smcvisms is for package lists in `debian/control` to be in alphabetical order.

Reply:
I ran `:sort` on this and squashed the result in.